credentialdigger/models/snippet_model/snippet_model.py

```diff
@@ -70,23 +68,29 @@ def analyze(self, discovery):
 
         # Convert data format
         data = self._pre_process(raw_data)
+        # Classify as a 'Leak' if this is a private key.
+        if(self._check_private_key(data)):
+            return False
+        if(not any(not c.isalnum() and c not in ' _.,?!/' for c in raw_data)):
```

Comment:
what is this check?

Reply:
As the comments below indicates, it is used to ignore snippets that look like regular phrases.
Example : 
```python
# Insert you password below
```
Will be ignored and not passed to the classifier. In other words, it filters out mostly comments and strings that get detected by the regex rules and are not containing password/token assignments.